app/src/main/java/org/p2p/wallet/auth/AuthModule.kt

```diff
@@ -145,8 +148,40 @@ object AuthModule {
                 durationTracker = get()
             )
         }
+        single(named(SMS_TIMER_ONBOARDING)) {
+            SmsInputTimer(get())
+        }
```

Comment:
```suggestion
        singleOf(::SmsInputTimer) { 
            named(SMS_TIMER_ONBOARDING)
       }
```

and I would rather use enum classes instead of strings

Reply:
fixed except enum, next time with on-ramp flow